src/markdown/sql/sql-limitations.md

```diff
@@ -272,9 +272,20 @@ This can produce results that are unexpected for those unfamiliar with the under
 - `MOD(numeric1, numeric2)`
 
 Note that using `%` as a modulo operator is not supported. Use `MOD` instead.
-### String Operators and Functions
-- `SUBSTRING(string FROM integer)` or `SUBSTRING(string, integer)`
-- `SUBSTRING(string FROM integer FOR integer)` or `SUBSTRING(string, integer, integer)`
+
+### String Operators and Functions  
+- `string || string` or `{fn CONCAT(string, string)}` to concatenate two strings.
```

Comment:
Also added the {fn ()} escape syntax for the string functions. Not sure it it looks a bit inconsistent now since we haven't done this yet for date functions.